.gitignore

```diff
@@ -1,4 +1,4 @@
-target
+/target
 Cargo.lock
```

Comment:
Could you also add a `/` prefix to `Cargo.lock`? I believe it has the same issue. Thank you :)

```console
$ fd -HI Cargo.lock
Cargo.lock
tests/testsuite/cargo_add/locked_unchanged/in/Cargo.lock
tests/testsuite/cargo_add/lockfile_updated/in/Cargo.lock
tests/testsuite/cargo_add/lockfile_updated/out/Cargo.lock
tests/testsuite/cargo_remove/gc_patch/out/Cargo.lock
tests/testsuite/cargo_remove/update_lock_file/in/Cargo.lock
tests/testsuite/cargo_remove/update_lock_file/out/Cargo.lock
```

Reply:
done!
I also added it to the `src/*` entries since they all seem to be referring to the root as well.